server/routes/user.js

```diff
@@ -19,7 +19,7 @@ router.post(
     }
     const { name, email } = req.body;
     try {
-      const user = new User({ name, email });
+      const user = new User({ name, email, experience: {} });
```

Comment:
I think this might conflict with my PR a bit. I can easily update it to include an empty experience object.